lesson26/src/js/login.js

```diff
@@ -1,12 +1,60 @@
 import * as validation from "./validation";
 import { togglePassword } from "./module/toggle-password";
+import { Chance } from "chance";
+const chance = new Chance();
 
 const submitButton  = document.querySelector('[data-id="submit_button"]');
+const userNameInput = document.getElementById('username');
+const passwordInput = document.getElementById('current-password');
+const togglePasswordButton = document.getElementById('js-toggle-password');
 
 submitButton.addEventListener("click", (e) => {
     e.preventDefault();
+    init();
 });
 
+const isRegisteredUser = (inputValues) => {
+    const userData = {
+        name:"Nakagawa",
+        email: "nakagawa@sample.com",
+        password: "N302aoe3"
+    }
+
+    return ( inputValues.userName === userData.name || userData.email ) && inputValues.password === userData.password;
```

Comment:
MUST
If your name and email are incorrect, but only your password is correct, you can log in.

Reply:
You are absolutely right! 
I made a basic mistake. Our apologies. It has been corrected.
https://github.com/nakagawamai/js-lesson/pull/31/commits/b3947a855184e2917b5abd6873cb1dcc819f7092